Products/Poi/browser/response.py

```diff
@@ -244,8 +244,10 @@ def transitions_for_display(self):
         transitions.append(dict(value='', label=PMF(u'No change'),
                                 checked="checked"))
         for tdef in wftool.getTransitionsFor(context):
+            label_value = tdef['title_or_id']
+            label_value = PMF(label_value)
```

Comment:
You are using `PMF` here, which is for the `plone` domain.
But in the workflow xml files you are setting the Poi domain.
So here it should be `_(label_value)` I think.